### ISSUE TITLE
Error 13000, API for identity not supported

### ISSUE THREAD

**Issue by lokkwanfoo** [OfficeDev/Office-Add-in-NodeJS-SSO#26]
I used the NodeJS SSO sample and continued to develop an application from there on out. In Word Online, the application works perfectly fine, but in the Word Client, no matter which version I use, I get error 13000 "API for identity is not supported for this add-in" when using the function getAccessTokenAsync even when I'm using the NodeJS SSO sample without any changes. 

I've tried 2016/2017/2018 versions of Word (Semi-annual, semi-annual targeted, monthly, Insider). Right now I am on version 1801 9001.2068 Insider and I still get the error, while the [documentation](https://docs.microsoft.com/nl-nl/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets?view=office-js) says using an Insider version should suffice.

What seems to be the issue? I'm able to provide samples of my manifest/code if needed, but I'm not sure where the problem lies in the first place.

In the readme.md, it says I need at least version 8605.1000. Is this the Monthly-channel (targeted), Monthly, Semi-annual Channel (Targeted), Semi-annual Channel or Insider version? In the [list for Word versions](https://docs.microsoft.com/nl-nl/officeupdates/update-history-office365-proplus-by-date) and this [list for all Insider versions](https://support.office.com/en-ie/article/update-history-for-office-insider-for-windows-desktop-64bbb317-972a-4933-8b82-cc866f0b067c#InsiderLevel=Insider), I can't seem to find version 8605.1000 while I do need to know which version it is to be able to downgrade.

**Comment by Rick-Kirkham** [comment]
Please see this overview of the SSO preview: [SSO in Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/develop/sso-in-office-add-ins). 
In the **Preview Status** section, you'll see this paragraph: 

> SSO requires Office 365 (the subscription version of Office, also called “Click to Run”). You should use the latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see Be an Office Insider. Please note that when a build graduates to the production semi-annual channel, support for preview features, including SSO, is turned off for that build.

So you need to be on the lastest build that is distributed in the Insider channel. As of today that seems to be 1903 | 11406.20002.

I'll update the article that you linked to and the readme to make this clearer.

**Comment by ghost** [comment]
![ubfi](https://user-images.githubusercontent.com/11733366/53003482-4eb36e00-342f-11e9-9323-d979544fdfdb.png)

![zxc](https://user-images.githubusercontent.com/11733366/53003484-51ae5e80-342f-11e9-9174-e9acef1d64f4.png)

Still doesn't seem to do the trick. Is it an absolute **must** to be in the Insider-fast program? I am on the version you provided.

Edit:

![image](https://user-images.githubusercontent.com/11733366/53008519-ddc58380-3439-11e9-8d3a-9b384ab756be.png)

Tried again with an Office Home edition, in Monthly-channel (targeted) and Insider version. Both also do not work.

**Comment by Rick-Kirkham** [comment]
@lokkwanfoo Yes. You _must_ be in the Insiders program and receive the latest build from the Insiders channel. The API is not supported on any other channel and it is turned off older versions, even if they were originally received from the Insiders channel.

**Comment by ghost** [comment]
But as you see in my previous post, I am on the latest version and in the Insiders program and I still get the error. 

I don’t know what other version I can try?

**Comment by Rick-Kirkham** [comment]
@lokkwanfoo There was a misunderstanding. I thought that when you asked "Is it an absolute must to be in the Insider-fast program?" you meant to imply that you are not in the Insider program. I will reopen this issue, but because it is not really an issue for this sample, as such, I'm going to transfer the issue to the repo for the Office JavaScript library.
